<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Visit</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/005-add.png</normaloff>icons/005-add.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Visit</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/005-add.png</normaloff>icons/005-add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>931</width>
        <height>506</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string> Patient : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="patient_comboBox">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>1</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>10000</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string> Doctor :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="doctor_comboBox">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>1</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string> Recovery Percentage : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="percentage_lineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="addvisit">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>  Add Visit </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="styleSheet">
            <string notr="true">QHeaderView::section{
text-align:center;
background:#CDCDCD;
padding:1px;
margin:0px;
color:#000;
border:1px solid #BABABA;
border-left-width:0;
font-size: 9pt;
}</string>
           </property>
           <column>
            <property name="text">
             <string>Patient Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Doctor Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Day</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Recovery Percent</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
